federation/pkg/kubefed/init/init_test.go

```diff
@@ -207,6 +208,8 @@ func TestInitFederation(t *testing.T) {
 		},
 	}
 
+	etcdImage := "gcr.io/google_containers/etcd:3.0.17"
```

Comment:
Should this have a test? It looks like you added the field to the struct but didn't add a test case.